test/System/IntegrationTests/IncrementingTraceTests.cs

```diff
@@ -1593,7 +1593,7 @@ public void ReplicatedToSomeExceptionToReplicated()
 
             TestFrameWork.AssertValue("mtcAID", new List<int>() { 0, 1}, astLiveRunner);
 
-            Assert.IsTrue(astLiveRunner.RuntimeCore.RuntimeData.CallsiteCache.Count == 1);
+            Assert.IsTrue(astLiveRunner.RuntimeCore.RuntimeData.CallsiteCache.Count == 2);
```

Comment:
The callsite cache now stores 2 entries since all properties are also converted into methods, namely `getter` methods, which also have associated callsites.